src/installer/pkg/projects/netcoreappRIDs.props

```diff
@@ -51,6 +46,10 @@
          in our runtime.json to enable others to provide them.  -->
```

Comment:
Where is the runtime.json that this talks about going to show up?

I am wondering whether this is still needed.

Reply:
FWIW, this code is here for ages. PR isn't inventing anything new.

Reply:
Yes, I know it has been there for ages. I am trying to understand what it is still used for and whether it is still needed. I am not able to able to find runtime.json that the comment talks about.